Accessories/ExpansionBoard_uInverter/README.md

```diff
@@ -0,0 +1,14 @@
+# uInverter Board
+
+The uInverter (micro-inverter) board is an accessory for the AMDC. The purpose of this board is to serve as a low-cost, micro-level prototype board to demonstrate three-phase current regulation using the AMDC. It uses a 12V DC supply to drive a three phase RL load circuit to emulate a motor.  
+The board features -  
+- Three phase AC current and DC bus voltage sensing feedback to AMDC.   
+- Test points to measure voltages at various locations in the supply as well as load circuits.   
+- BNC connector to directly measure the current waveforms on an oscilloscope conveniently.    
+
+
+![PCB 3D](uInverter3D_02.JPG)
```

Comment:
Can you move this jpg file into the `docs/images/` folder along with the other images? This cleans up the root folder.